lib/services.identity.js

```diff
@@ -669,37 +669,9 @@ function updateIdentity(req, res, next) {
       brIdentity.setRoles(req.user.identity, identity, callback);
     }],
     update: ['updateRoles', function(callback, results) {
-      var identity = {id: results.getId};
-
-      // conditional values only set if preset
-      if(req.body.description) {
-        identity.description = req.body.description;
-      }
-      if(req.body.email) {
```

Comment:
Yeah, I'd think we'd want to do something special w/email like switch verification status off/on. Let's not include that for now.